contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -58,27 +59,23 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
         public receiptQueues;
 
     // Tracks the message hash and fee information for each message sent that has yet to be acknowledged
-    // with a receipt. The messages are tracked per chain and keyed by message ID.
-    // The first key is the blockchain ID of the destination chain, the second key is the message ID, and the value is the info
-    // for the uniquely identified message.
-    mapping(
-        bytes32 destinationBlockchainID => mapping(uint256 messageID => SentMessageInfo messageInfo)
-    ) public sentMessageInfo;
+    // with a receipt.
+    // The key is the message ID, and the value is the info for the uniquely identified message.
+    mapping(bytes32 messageID => SentMessageInfo messageInfo) public sentMessageInfo;
 
-    // Tracks the hash of messages that have been received but whose execution has never succeeded.
+    // Tracks the hash of messages that have been received but have never succeeded in execution.
     // Enables retrying of failed messages with higher gas limits. Message execution is guaranteed to
-    // succeed at most once. The first key is the blockchain ID of the sending chain, the second key is the message ID, and
-    // the value is the hash of the uniquely identified message whose execution failed.
-    mapping(bytes32 sourceBlockchainID => mapping(uint256 messageID => bytes32 messageHash)) public
-        receivedFailedMessageHashes;
-
-    // Tracks the relayer reward address for each message delivered from a given chain.
-    // Note that these values are also used to determine if a given message has been delivered or not.
-    // The first key is the blockchain ID, the second key is the message ID, and the value is the reward address
-    // provided by the deliverer of the uniquely identified message.
-    mapping(
-        bytes32 sourceBlockchainID => mapping(uint256 messageID => address relayerRewardAddress)
-    ) internal _relayerRewardAddresses;
+    // succeed at most once. The key is the message ID, and the value is the hash of the uniquely
+    // identified message whose execution failed.
+    mapping(bytes32 messageID => bytes32 messageHash) public receivedFailedMessageHashes;
+
+    // Tracks the message nonce for each message that has been received.
+    // Note: the `messageNonce` values are also used to determine if a given message has been received or not.
```

Comment:
Can you explicitly note the key and value, as is the case with the other mappings?